lib/parliament/request.rb

```diff
@@ -108,7 +108,12 @@ def get(params: nil)
       endpoint_uri = URI.parse(api_endpoint)
       endpoint_uri.query = URI.encode_www_form(params.to_a) unless params.nil?
 
-      net_response = Net::HTTP.get_response(URI(endpoint_uri))
+      request = Net::HTTP::Get.new(URI(endpoint_uri))
```

Comment:
Do we need to wrap this in an explicit URI call? as we are parsing the string into a uri above?